.gitlab-ci.yml

```diff
@@ -27,12 +29,14 @@ test:lint:
     - master
```

Comment:
here is an `except` definition in the job already, unfortunately they won't get combined automatically

Reply:
Ops, sorry, I've missed it. Anyway, this change is not for the "staging" pipeline, it is for the `saas-*` tag which is used to re-tag the staging_SHA docker image to `saas-*`, thus no tests are expected to run because we are not running (again) docker-build.

Reply:
that's what I meant, as we won't tag unless the staging build is successful/ the staging image is available - I should have been more explicit with my remark 😅